server-kotlin/src/test/kotlin/digital/cesko/city_search/CitySearchTest.kt

```diff
@@ -23,6 +24,7 @@ class CitySearchTest : AbstractSpringTest() {
     fun testSearchKnownCity() {
         get("/api/v2/service/citysearch?query=Cernosice").andExpect {
             status { isOk }
+            header { string(HttpHeaders.LINK, "<https://www.cityvizor.cz/api/public/profiles/3/avatar>; rel=\"preload\"; as=\"image\"") }
```

Comment:
V tehle oblasti se neorientuji, ale nejde to nejak lip resit na klientovi?

Reply:
da se vyrenderovat ekvivalentni <link ..> tag a nebo ruzne ohacky pres skryte divy s backgroundem. Mozna nekdo dalsi poradi.

Reply:
FYI zkousim to na klientovi